src/shared/components/input/Input.scss

```diff
@@ -1,23 +1,63 @@
 @import 'src/styles/common';
 
 .input {
-  display: inline-block;
-  width: 100%;
+  &::placeholder {
+    color: $dark-grey;
+    font-weight: $light;
+  }
 
   &:focus,
   &:active {
     outline: none;
   }
+}
 
-  &::placeholder {
-    color: $black;
-    font-weight: $light;
+.rightCorner {
+  display: flex;
+  align-items: center;
+  flex: 0 0 auto;
+}
+
+.inputWrapper {
+  background-color: var(--input-background-colour, $white);
+  display: inline-flex;
+  padding: var(--input-padding, 0 15px);
+  width: var(--input-width, 100%);
```

Comment:
Unsure about both the purpose of this class and the names of these variables. Something to discuss...